spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -688,23 +688,23 @@ public void invalidDataImport() {
 
     createIcebergTable("id Integer, name String, dept String, subdept String");
 
-    AssertHelpers.assertThrows(
-        "Should forbid adding of partitioned data to unpartitioned table",
-        IllegalArgumentException.class,
-        "Cannot use partition filter with an unpartitioned table",
-        () ->
-            scalarSql(
-                "CALL %s.system.add_files('%s', '`parquet`.`%s`', map('id', 1))",
-                catalogName, tableName, fileTableDir.getAbsolutePath()));
-
-    AssertHelpers.assertThrows(
-        "Should forbid adding of partitioned data to unpartitioned table",
-        IllegalArgumentException.class,
-        "Cannot add partitioned files to an unpartitioned table",
-        () ->
-            scalarSql(
-                "CALL %s.system.add_files('%s', '`parquet`.`%s`')",
-                catalogName, tableName, fileTableDir.getAbsolutePath()));
+    Assertions.assertThatThrownBy(
```

Comment:
it should be fine to statically import the method here, similar to how `assertThat` is already statically imported in this class.

Reply:
The code was changed mechanically, so thanks for catching that.
I know that static imports are generally discouraged in the project, so is this comment about this source file in particular, or more broadly about `Assertions.*`?

Reply:
the comment is only about statically importing methods from `Assertions`, but I guess it would be too much work now to statically import `assertThatThrownBy`, so let's leave it as-is

Reply:
If we have consensu that `Assertions` should be used with static imports, do you think it could be helpful to support this with static code analysis?

Reply:
we could eventually support this via static code analysis but atm we have a mix of static vs non-static usage wrt Assertions